markdown/3.x/en/api/ff-communication.api.md

```diff
@@ -7,9 +7,9 @@ ___
 | **version**&nbsp;(String)&nbsp;(default: 7.2) | Your FACT-Finder version. Only major and minor version like "7.2" |
 | **api**&nbsp;(String)&nbsp;(default: `latest`) | Only relevant when `version="ng"`. The version of your FACT-Finder NG REST API. Possible values are `v2`, `v3` and `latest`. It is recommended to use an explicit version instead of `latest`. This avoids unintentionally breaking your application by updating your Web Components reference when a new API version becomes available. |
 | **channel**&nbsp;(String)&nbsp;(default: empty) | Your channel name. Has to be the same as the channel name configured in the FACT-Finder backend. |
-| **search-immediate**&nbsp;(Boolean) | If this property is present, FACT-Finder Web Components will start searching as soon as they are loaded. |
+| **search-immediate**&nbsp;(Boolean) | If this property is present, FACT-Finder Web Components will start searching as soon as they are loaded. For this to work, a `default-query` needs to be set with correct value (see `default-query` description) or should not be set at all. |
```

Comment:
Placing the reference to `default-query` at the end might allow a smoother reading flow.
```suggestion
| **search-immediate**&nbsp;(Boolean) | If this property is present, FACT-Finder Web Components will start searching as soon as they are loaded. For this to work, a `default-query` needs to be set with a correct value or to be not set at all (see `default-query` description). |
```